File: 프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.py
def solution(s):
    answer = 0
    box = []
    s_split = s.split(" ")
    for i in s_split:
        if i != "Z":
            answer += int(i)
            box.append(int(i))

        else:
            answer -= box[-1]

    
    return answer

File: 프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.java
import java.util.Arrays;
class Solution {
    public int solution(String s) {
    int answer = 0;
    return Arrays.stream(s.replace(" Z", "Z").split(" "))
                    .filter(i -> i.matches("-?\\d+"))
                    .mapToInt(Integer::parseInt)
                    .sum();
}
}